llvm/docs/ReleaseNotes.md

```diff
@@ -61,6 +61,8 @@ Changes to the LLVM IR
 Changes to LLVM infrastructure
 ------------------------------
 
+* The `TargetIntrinsicInfo` support has been removed.
```

Comment:
Note sounds ambiguous to the unfamiliar

Reply:
The commit that added this has the description: "add support for target intrinsics being defined in the
target directories themselves". I can word this as: 

Removed support for target intrinsics being defined in the target directories themselves (i.e., the `TargetIntrinsicInfo` class).

Reply:
Done.